### DISCUSSION TITLE
Prompt for input of vars

### DISCUSSION THREAD

**Discussion by scflode** [usebruno/bruno#525] [Ideas]
Hi,

just wanted to check if it would be feasible (maybe already planned) to introduce prompts for inputs (like Insomnia has). This would make using endpoints much easier by f.e. having default values preset but overwritable for different use cases.

With Insomnia this works that before the request is sent you are presented an input with a default value preset and can change it. F.e. an ID or some other use case specific data. This avoids having to change the "code" which then might accidentally be committed.

I see that there is a lot of movement and work being done so it's just an idea for future development.

**Comment by AnderssonPeter** [comment]
Fellow Insomnia refuge here and this is one of the main features I miss in Bruno, are there any plans to add it?

I tried to use the javascript function `prompt` in a pre request script but it just complains that that function does not exist.

**Reply by cmoine-swi** [reply]
Similar to https://github.com/usebruno/bruno/issues/1033
